examples/stix.py

```diff
@@ -8,26 +8,27 @@
 """
 
 import pickle
-import urllib.request
 
 import astropy.units as apu
 import matplotlib.pyplot as plt
+import numpy as np
 
 from xrayvision.clean import vis_clean
 from xrayvision.imaging import vis_psf_map, vis_to_map
-from xrayvision.mem import mem
+from xrayvision.mem import mem, resistant_mean
 
 ###############################################################################
 # Create images from STIX visibility data.
 #
 # The STIX data has already been prepared and stored in python pickle format
 # the variables can be simply restored.
 
-stix_data = pickle.load(urllib.request.urlopen("https://pub099.cs.technik.fhnw.ch/demo/stix_vis.pkl"))
+with open("./stix_vis.pkl", "rb") as file:
+    stix_data = pickle.load(file)
```

Comment:
Ok I gess stix_vis.pkl has been updated? Let use this for the moment. I think what we should do is only store the data in the file and create the visibility here that way it won't change if we change the vis object.

Reply:
I agree, we should save the data in a file, read it and build the visibility structure

Reply:
Open #69 to deal with this later